src/Libraries/RevitNodes/Elements/Element.cs

```diff
@@ -173,6 +173,21 @@ public Category GetCategory
         }
 
         /// <summary>
+        /// Get Element Type
+        /// </summary>
+        public Element ElementType
+        {
+            get
+            {
+                // Get the active Document
+                Autodesk.Revit.DB.Document document = DocumentManager.Instance.CurrentDBDocument;
+
+                return document.GetElement(this.InternalElement.GetTypeId()).ToDSType(true);
```

Comment:
this works: 
![image](https://cloud.githubusercontent.com/assets/529781/18885477/ce67950e-84b9-11e6-8041-1e4e4a22b250.png)